packages/@okta/i18n/src/properties/login_ja.properties

```diff
@@ -567,7 +569,7 @@ password.complexity.symbol.description = \u8A18\u53F7
 password.complexity.no_username.description = \u30E6\u30FC\u30B6\u30FC\u540D\u306E\u4E00\u90E8\u307E\u305F\u306F\u5168\u90E8\u306E\u4F7F\u7528\u4E0D\u53EF
 password.complexity.no_first_name.description = \u304A\u540D\u524D\u3092\u4F7F\u7528\u3059\u308B\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093
 password.complexity.no_last_name.description = \u82D7\u5B57\u3092\u4F7F\u7528\u3059\u308B\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093
-password.complexity.history.description = \u904E\u53BB\u306B\u4F7F\u3063\u305F\u3053\u3068\u306E\u3042\u308B{0}\u4E16\u4EE3\u524D\u307E\u3067\u306E\u30D1\u30B9\u30EF\u30FC\u30C9\u306F\u5229\u7528\u3067\u304D\u307E\u305B\u3093
+password.complexity.history.description = \u904E\u53BB\u306B\u4F7F\u3063\u305F\u3053\u3068\u306E\u3042\u308B{0}\u4E16\u4EE3\u524D\u307E\u3067\u306E\u30D1\u30B9\u30EF\u30FC\u30C9\u306F\u5229\u7528\u3067\u304D\u307E\u305B\u3093\u3002
```

Comment:
@aarongranick-okta We need a change here. Should we amend this file or submit another commit? What's your team's naming convention for updates? @santhoshbalakrishnan